frontend/src/lib/Course.svelte

```diff
@@ -32,20 +33,32 @@
 </script>
 
 <div>
-  <h3 class="text-sm font-bold">
-    <span title={data.subjectDescription}>{data.subject}</span>
-    {data.catalogNumber}:
-    {data.title || "[No Title]"} ({data.semester})
-  </h3>
-  <p class="text-sm mb-1">
-    {#each data.instructors as instructor, i}
-      <a
-        href={instructor.email
-          ? `mailto:${instructor.email.toLowerCase()}`
-          : undefined}>{instructor.name}</a
-      >{#if i < data.instructors.length - 1}{", "}{/if}
-    {/each}
-  </p>
+  <div class="flex gap-2">
```

Comment:
I think this could do with a much bigger gap actually. Here's `gap-8`:
![screenshot-gap-8](https://github.com/ekzhang/classes.wtf/assets/16979554/271ccba9-17b3-4a11-b435-690ddec285c7)